okhttp-logging-interceptor/src/main/java/com/squareup/okhttp/logging/HttpLoggingInterceptor.java

```diff
@@ -145,7 +149,7 @@ public void setLevel(Level level) {
     Connection connection = chain.connection();
     Protocol protocol = connection != null ? connection.getProtocol() : Protocol.HTTP_1_1;
     String requestStartMessage =
-        "--> " + request.method() + ' ' + requestPath(request.httpUrl()) + ' ' + protocol(protocol);
+        "--> " + request.method() + ' ' + requestUrl(request.httpUrl()) + ' ' + protocol(protocol);
```

Comment:
You can just eliminate the method call altogether. The string concatenation will take care of calling `.toString()`.